.github/workflows/main.yml

```diff
@@ -287,7 +287,7 @@ jobs:
       ANT_OPTS: -Dmetabuild.jsonurl=https://raw.githubusercontent.com/apache/netbeans-jenkins-lib/master/meta/netbeansrelease.json
     strategy:
       matrix:
-        java: [ '17' ]
+        java: [ '23' ]
```

Comment:
I hope this doesn't cause issues, but we will see next time this PR fully builds and all paperwork checks run, I removed the no-build label now.

Reply:
this seems to work. But I might change it back anyway. Probably not worth the risk to run sig test-tests on JDK 23.